### ISSUE TITLE
ESP32-S3-WROOM2

### ISSUE THREAD

**Issue by higaski** [espressif/kicad-libraries#49]
Any chance we can expect the WROOM2 versions of the S3 modules any time soon? From what I've seen they have slightly rearranged ground pads in the middle compared to the WROOM1.

**Comment by pedrominatel** [comment]
Hi @higaski,

The [ESP32-S3-WROOM-2](https://www.espressif.com/sites/default/files/documentation/esp32-s3-wroom-2_datasheet_en.pdf) will be added according to the datasheet landing pattern.

![image](https://user-images.githubusercontent.com/274658/178227550-530bc6fa-3f76-4b8f-9bf2-279d146138a4.png)

**Comment by higaski** [comment]
Ok, let me rephrase my question.

In case I need a footprint of this part within the next 2-3 weeks, should I better draw it myself or can I count on you guys doing it. :stuck_out_tongue_closed_eyes:

**Comment by pedrominatel** [comment]
We can manage that in the next week or soon.

**Comment by pedrominatel** [comment]
Done, @higaski. PTAL
